src/main/java/net/neoforged/moddevgradle/internal/ModDevPlugin.java

```diff
@@ -147,6 +150,10 @@ public void apply(Project project) {
                 content.includeModule("net.neoforged", "minecraft-dependencies");
             });
         });
+        repositories.addLast(repositories.maven(repo -> {
```

Comment:
Why addLast?

Reply:
If you do `repositories.maven(...)` it should _already_ have just been added at the end.

Reply:
I just copied the code from one place to another.

Reply:
I know but can you fix it anyway? :P